tap_mailchimp/schemas/campaigns.json

```diff
@@ -453,6 +464,360 @@
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "from_name": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "reply_to": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "content_description": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "recipients": {
+              "type": [
+                "null",
+                "string"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "rss_opts": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "feed_url": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "frequency": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "schedule": {
+          "type": [
+            "null",
+            "object"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "hour": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "daily_send": {
+              "type": [
+                "null",
+                "object"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "sunday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "monday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "tuesday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "wednesday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "thursday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "friday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "saturday": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                }
+              }
+            },
+            "weekly_send_day": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "monthly_send_Date": {
+              "type": [
+                "null",
+                "string"
```

Comment:
The datatype is the number for this field. Can you please verify?

Reply:
Updated the datatype to number.